index.js

```diff
@@ -22,6 +26,11 @@ module.exports = postcss.plugin("postcss-calc", function(options) {
       helpers.try(function transformCSSCalc() {
         var value = reduceCSSCalc(decl.value, precision)
 
+        if (throwWhenCannotReduce && CONTAINS_CALC.test(value)) {
+          throw decl.error("Could not reduce expression: " + decl.value,
```

Comment:
Example of usage https://github.com/postcss/postcss-custom-properties/blob/master/index.js#L56-L59
You will need to add "result" as a second parameter here https://github.com/thejameskyle/postcss-calc/blob/2ca864fa70ce55956c2f5c5d3b1f0daee993786f/index.js#L19.
If you do that, please rename the option to "warnWhenCannotResolve" (since in this case it's full resolution that we are testing), add a note in the top of the CHANGELOG (something like `Added: "warnWhenCannotResolve" option to warn when calc() are not reduced to a single value`) and document it in the README and we should be good to go :+1: